packages/react-instantsearch-core/src/connectors/connectMenu.js

```diff
@@ -99,8 +99,11 @@ export default createConnector({
     meta,
     searchForFacetValuesResults
   ) {
-    const { attribute, searchable } = props;
-    const results = getResults(searchResults, { ais: props.contextValue });
+    const { attribute, searchable, indexContextValue } = props;
+    const results = getResults(searchResults, {
+      ais: props.contextValue,
+      multiIndexContext: props.indexContextValue,
```

Comment:
Since we destruct the value above we can re-use it. We can apply the change in the all function.

Reply:
I didn't do that, because grepping the code for this pattern would become harder, since we plan on removing this in favour of a nicer API